### PR TITLE
Update organising_code.md to cover optional arguments

### PR DESCRIPTION
**Pull request by Beliavsky**
The optional arguments feature is described, and the vector_norm function is updated to have an optional argument. A main program shows how vector_norm can be called with and without the optional argument.

### CODE REVIEW

File: source/learn/quickstart/variables.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ program variables
   implicit none
 
   integer :: amount
-  real :: pi
+  real :: pi, e ! two variables declared
```
Comment (perazz) [inline source/learn/quickstart/variables.md:42]:
```suggestion
  real :: pi, e ! two `real` variables declared
```

**Review by perazz — APPROVED** [review]
LGTM, thank you @Beliavsky

**Review by henilp105 — APPROVED** [review]
Thanks @Beliavsky , LGTM.

**Comment by henilp105** [comment]
Merging

**Pull request merged**